ldms/src/store/influx/store_influx.c

```diff
@@ -64,7 +64,6 @@ static ovis_log_t mylog;
 static char host_port[64];	/* hostname:port_no for influxdb */
 struct influx_store {
 	struct ldmsd_store *store;
```

Comment:
Lets just remove "struct ldmsd_store *store" from struct influx_store. The only place that "store" is used, it is used in error. Better to just get rid of it.